corehq/apps/appstore/static/appstore/js/facet_sidebar.js

```diff
@@ -88,4 +97,4 @@ $(function () {
         e.preventDefault();
         $update_btn.click();
     });
-});
\ No newline at end of file
+});
```

Comment:
Odd that this file lives in appstore when it's used outside of the exchange. Maybe hqwebapp would be better?

(Not blocking, I agree it makes sense to limit this PR to B3-necessary changes, but it would be unfortunate to review this large a chunk of code and not comment on possible future improvements.)